pom.xml

```diff
@@ -106,6 +106,23 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-security</artifactId>
         </dependency>
+        <dependency>
```

Comment:
You could define a custom property under <properties> and refer to it from your dependency. Preferred way is to place the property in parent pom (if exist and is a multi module project) (in your case placing the version in properties is enough).
Example:
`<properties>
<http.client.version>4.3.6</http.client.version>
</properties>
...
<dependencies>
   <dependency>
     <groupId>org.apache.httpcomponents</groupId>
     <artifactId>httpclient</artifactId>
     <version>${http.client.version}</version>
   </dependency>
</dependencies>`

Additionaly nice article in the subject: https://medium.com/@mahibabar/centralized-dependencies-version-management-using-maven-1dba05493719